index.js

```diff
@@ -42,28 +47,36 @@ const parseEvent = (event) => {
 };
 
 // Prepare the Messages and Options
-const prepareLogs = (eventData) => {
+const prepareLogs = (eventData, raw_event_message) => {
     return eventData.logEvents.map((event) => {
-        return {
-            line: JSON.stringify({
-                message: event.message
-                , source: 'cloudwatch'
-                , event: {
-                    type: eventData.messageType
-                    , id: event.id
-                }
-                , log: {
-                    group: eventData.logGroup
-                    , stream: eventData.logStream
-                }
-            })
-            , timestamp: event.timestamp
+        const eventMetadata = {
+            source: 'cloudwatch'
+            , event: {
+                type: eventData.messageType
+                , id: event.id
+            }, log: {
+                group: eventData.logGroup
+                , stream: eventData.logStream
+            }
+        };
+
+        const eventLog = {
+            timestamp: event.timestamp
             , file: eventData.logStream
             , meta: {
                 owner: eventData.owner
                 , filters: eventData.subscriptionFilters
-            }
+            }, line: JSON.stringify(Object.assign({}, {
+                message: event.message
+            }, eventMetadata))
         };
+
+        if (raw_event_message) {
+            eventLog.line = event.message;
+            eventLog.meta = Object.assign({}, eventLog.meta, eventMetadata);
+        }
```

Comment:
I don't think the changelog or documentation clearly explains this behavior change.

So normally the eventMetaData is stringified and coupled in the line with the message.

With `RAW_EVENT_MESSAGE` enabled we appear to change this behavior and instead keep the raw message in the line field and then merge the eventMetaData into the meta field.

Reply:
Added one more brief explanation to the `README` - looking for suggestions